RTClib.cpp

```diff
@@ -1215,7 +1215,10 @@ Ds3231SqwPinMode RTC_DS3231::readSqwPinMode() {
   Wire.requestFrom((uint8_t)DS3231_ADDRESS, (uint8_t)1);
   mode = Wire._I2C_READ();
 
-  mode &= 0x93;
+  mode &= 0x1C;
+  if (mode & 0x04) {
```

Comment:
If the header is updated with the following, then I think the special-case handling of DS3231_OFF in readSqwPinMode and writeSqwPinMode could be eliminated.

```
enum Ds3231SqwPinMode {
  DS3231_OFF = 0x01, 
  ...
```